README.md

```diff
@@ -45,15 +45,10 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/HAMMAS-SALEEM)
+- Twitter: [@twitterhandle](https://twitter.com/HammasSaleem4)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/hammas-saleem-407)
```

Comment:
- [ ] Please remember to change the handles to match your own! That way your README looks more professional!

Reply:
Sir,
     First of all, I would like to give appreciations to you to give a review of my code. But I want to say that I have my .gitignore file in my repo in my feature branch but I did not merge them yet and the .gitignore file is indicating which files should be ignored. Also, in some previous commits, I deleted the file test.md but then I added it in my project in screenshots below you can see. Pardon, I forgot to commit when I added it again but it is in my project you can see in screenshots below. You can match gitignore file in my repo in the feature branch with my commits which are after the test.md delete. You will see this file there and the code there. Now, i'm working on other changes so if you see any commit so you can verify i am working on other things.

![01](https://user-images.githubusercontent.com/93039359/143066580-d0a7a653-a0ad-427e-bc48-445f9449bc32.PNG)
![02](https://user-images.githubusercontent.com/93039359/143066601-bcfbfe99-1f17-4dfe-b2e4-179bb3c45cf7.PNG)
![03](https://user-images.githubusercontent.com/93039359/143066605-9cc3d63a-3ee9-4e03-9dd8-811ce9a4113c.PNG)
:

Reply:
Yes, don't worry about it! I just saw your commit history but did not check your file so the .gitignore is good to go!